src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -62,21 +63,62 @@ This page lists all the commands and options available under the [web-ext](https
 <article class="module-content grid-x grid-padding-x">
 <div class="cell small-12">
 
+## What's changed in Version 8
+
+Released in May 2024, the main change in version 8 of `web-ext` is that `web-ext sign` now creates a listing for an extension not previously listed on [addons.mozilla.org](https://addons.mozilla.org) (AMO) by default. This feature was previewed in version 7 with the [--use-submission-api](#use-submission-api) option, which is now removed. This feature is achieved using the submission features of [addons.mozilla.org add-on API v5](https://addons-server.readthedocs.io/en/latest/topics/api/addons.html).
+
+### Removed
+
+These version 7 `web-ext sign` options are removed:
+
+- [`--use-submission-api`](#use-submission-api)
+- [`--api-url-prefix`](#api-url-prefix)
+- --disable-progress-bar (undocumented feature)
+
+### Updates
+
+These `web-ext sign` options have changed:
+
+- [`--amo-base-url`](#amo-base-url) no longer requires `use-submission-api` to be set.
+- [`--channel`](#channel) is now required.
+
+### Additions
+
+These features are added:
+
+- [`web-ext dump-config`](#web-ext-dump-config), this new command prints a copy of the configuration data to the terminal.
+- [`web-ext sign --approval-timeout`](#approval-timeout) enables this number of milliseconds to wait for approval before giving up to be set.
+- [`web-ext sign --upload-source-code`](#upload-source-code) enables a file containing human-readable source code to be uploaded.
+
+</div>
+</article>
+</section>
+
+<!-- END: Single Column Body Module -->
+
+
+<!-- Single Column Body Module -->
+
+<section id="commands" class="module">
+
+<article class="module-content grid-x grid-padding-x">
+<div class="cell small-12">
+
 ## Commands
 
-web-ext has the following commands; options specific to these commands are included as subsections.
+web-ext has these commands; a command's options are included as subsections.
```

Comment:
Looks like we consistently style the name of the tool in monospace on this page.

```suggestion
`web-ext` has these commands; a command's options are included as subsections.
```